drivers/sensor/apds9960/Kconfig

```diff
@@ -23,15 +23,22 @@ config APDS9960_TRIGGER_NONE
 
 config APDS9960_TRIGGER_GLOBAL_THREAD
 	bool "Use global thread"
-	depends on GPIO
-	depends on $(dt_compat_any_has_prop,$(DT_COMPAT_AVAGO_APDS9960),int-gpios)
 	select APDS9960_TRIGGER
+	select APDS9960_INTERRUPT_PIN
```

Comment:
No need to select it from here. It already defaults to y if it has the GPIO property.